README.md

```diff
@@ -1,20 +1,18 @@
 <p align="center">
-  <img src="https://avatars.githubusercontent.com/u/96310119" alt="PolyMC logo"/>
+  <img src="/program_info/polymc-logo.png#gh-light-mode-only" alt="PolyMC logo"/>
```

Comment:
Why not store this in githubusercontent, or on a separate repo?

Reply:
I wasn't sure how we where storing icons etc the program_info seemed to say thats where we would store branding. I am mistaken.

Reply:
Ok, move it to polymc-{dark,light}.png and it's fine to keep it in program_info